server.js

```diff
@@ -1,188 +1,149 @@
 // Constants
-var WALK_SPEED = 4;
-var FPS = 30;
+const WALK_SPEED = 4;
+const FPS = 30;
 
 // Globals
-var id = 0;
-var map;
-var mapSize = {x: 0, y: 0};
-
-var express = require('express');
-var app = express.createServer()
-var io = require('socket.io').listen(app);
-var fs = require('fs');
-var _ = require('./public/underscore.js')._;
-eval(fs.readFileSync('public/common.js') + '');
-
-
-io.set('log level', 1); // Reduce the log messages
-
-app.configure(function() {
-  app.use(express.static(__dirname + '/public'));
-});
-
-app.listen(8080);
-
-app.get('/', function (req, res) {
-    res.sendfile(__dirname + '/index.html');
+let id = 0;
+let map;
+let mapSize = { x: 0, y: 0 };
+let entities = [];
+
+const express = require('express');
+const app = express();
+const http = require('http').Server(app);
+const io = require('socket.io')(http);
+const fs = require('fs');
+const _ = require('./public/underscore.js')._;
+
+fs.readFile('public/common.js', 'utf8', (err, data) => {
+    if (err) throw err;
```

Comment:
Let's not use short-hand functions.
```suggestion
    if (err) {
        throw err;
    }
```

Reply:
Done.